lib/rucio/db/sqla/util.py

```diff
@@ -201,46 +204,48 @@ def create_root_account(create_counters=True):
     iaa4 = models.IdentityAccountAssociation(identity=identity4.identity, identity_type=identity4.identity_type, account=account.account, is_default=True)
 
     # Account counters
-    if create_counters:
-        create_counters_for_new_account(account=account.account, session=s)
-
-    # Apply
-    for identity in [identity1, identity2, identity3, identity4]:
-        try:
-            s.add(identity)
-            s.commit()
-        except IntegrityError:
-            # Identities may already be in the DB when running multi-VO conversion
-            s.rollback()
-    s.add(account)
-    s.commit()
-    s.add_all([iaa1, iaa2, iaa3, iaa4])
-    s.commit()
+    with session_scoped() as s:
+        s.begin()
+        if create_counters:
+            create_counters_for_new_account(account=account.account, session=s)
+
+        # Apply
+        for identity in [identity1, identity2, identity3, identity4]:
+            try:
+                s.add(identity)
+                s.commit()
+            except IntegrityError:
+                # Identities may already be in the DB when running multi-VO conversion
+                s.rollback()
+        s.add(account)
+        s.flush()
+        s.add_all([iaa1, iaa2, iaa3, iaa4])
+        s.commit()
 
 
 def get_db_time():
     """ Gives the utc time on the db. """
-    s = get_session()
+    session_scoped = get_session()
     try:
         storage_date_format = None
-        if s.bind.dialect.name == 'oracle':
+        if session_scoped.bind.dialect.name == 'oracle':
             query = select([text("sys_extract_utc(systimestamp)")])
-        elif s.bind.dialect.name == 'mysql':
+        elif session_scoped.bind.dialect.name == 'mysql':
             query = select([text("utc_timestamp()")])
-        elif s.bind.dialect.name == 'sqlite':
+        elif session_scoped.bind.dialect.name == 'sqlite':
             query = select([text("datetime('now', 'utc')")])
             storage_date_format = '%Y-%m-%d  %H:%M:%S'
         else:
             query = select([func.current_date()])
 
-        for now, in s.execute(query):
-            if storage_date_format:
-                return datetime.strptime(now, storage_date_format)
-            return now
-
+        session = session_scoped()
+            session.begin():
+            for now, in session.execute(query):
+                if storage_date_format:
+                    return datetime.strptime(now, storage_date_format)
+                return now
     finally:
-        s.remove()
-
+        session_scope.remove()
```

Comment:
session_scoped (missed the 'd' at the end)

Reply:
Thanks for catching the typo.